tests/test_client_session.py

```diff
@@ -706,3 +708,13 @@ async def test_requote_redirect_url_default_disable() -> None:
     session = ClientSession(requote_redirect_url=False)
     assert not session.requote_redirect_url
     await session.close()
+
+
+async def test_http_GET_with_params_argument(session_params: Any) -> None:
+    try:
+        async with session_params as session:
+            response: aiohttp.ClientResponse = await session.get("http://example.com")
+
+            assert response.url, f"http://example.com/x=2"
+    except Exception:
+        pass
```

Comment:
It's a very bad idea to ignore errors in tests. This way you don't test anything at all.